examples/werkstatt/README.md

```
@@ -0,0 +1,66 @@
+  }
+
+  paramChanged(label, value) {
+    // Optional: called when a @param knob changes
+    // Store the value for use in process()
+  }
+}
+```
+
+## @param Format
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)